### DISCUSSION TITLE
Showcase projects using this API

### DISCUSSION THREAD

**Discussion by ByMykel** [ByMykel/CSGO-API#112] [Show and tell]
So, as the title suggests, I'm curious about the projects that are using this API. I would love for you to share your projects with me, and also mention anything you feel is missing in this API.

**Comment by ByMykel** [comment]
I'll start. 

I have a simple project where I list items and their prices. The purpose was to search items fast and have a simple UI

https://bymykel.github.io/counter-strike-items/#/

![image](https://github.com/user-attachments/assets/7cdc43c0-ed80-4a1f-8b91-66c3cf992b76)

**Comment by cs7market** [comment]
Hello. CS7.market use this project to fill the starting items on our market. Thank you very much for the work done

**Comment by piyanggoon** [comment]
![CleanShot 2567-03-24 at 12 09 26@2x](https://github.com/ByMykel/CSGO-API/assets/31122178/ec4e941b-118a-42f6-8537-b48036d585de)

Working in process...

**Comment by leonlarsson** [comment]
I have made a little case opening simulator using data from the API: https://case-sim.leonlarsson.com/
https://github.com/leonlarsson/cs-case-sim

**Comment by MeckeDev** [comment]
Hey ByMykel   
I really wanna thank you for keeping this always up to date, we developed [this Website](https://www.inspect.tools/) with your API, you can generate inspect urls and gen codes for every item in the game, even if they don't exist, like this AWP | Heat Treated
![image](https://github.com/user-attachments/assets/cd428657-55b9-4e9f-8c50-550139c43c0d)

**Comment by ShoggyR79** [comment]
Hey @ByMykel, I randomly discovered this API after my last API source got sold to a gambling website.. I own cs2builder.com and hopefully I can use this one now! do you happen to know if there is any plan in the future to include last updated price?

**Reply by ByMykel** [reply]
Hi, I am not as this is just a static JSON that I update every time theres an update. For prices you can check my other repo https://raw.githubusercontent.com/ByMykel/counter-strike-price-tracker/refs/heads/main/static/prices/latest.json (prices in EUR)

**Reply by ShoggyR79** [reply]
is there an API i can use to get this file? could you give me some suggestion on how to best query this price? I do price update on a daily cadence

**Reply by ByMykel** [reply]
No, there isn’t. What I do in other projects is fetch that whole object once and use it for all the items like [here](https://bymykel.github.io/counter-strike-items/#/).

I'm not sure what could be a better way to fetch this data. Maybe you could fork the [project](https://github.com/ByMykel/counter-strike-price-tracker/tree/main/static) and do some changes to fake an API like I did with the items price history.

I save a single file for every item price history. The name is the sha1 of the marketHashName so I got something like this:

[/pricehistory/18fd71c0bc271864402b442ea7fd6650223159fe.json](https://raw.githubusercontent.com/ByMykel/counter-strike-price-tracker/main/static/pricehistory/18fd71c0bc271864402b442ea7fd6650223159fe.json)
[/pricehistory/c8f692b9eaeef144031aaf78d7ff60ecbe289dec.json](https://raw.githubusercontent.com/ByMykel/counter-strike-price-tracker/main/static/pricehistory/c8f692b9eaeef144031aaf78d7ff60ecbe289dec.json)
[/pricehistory/6aea523ef0fce5b5b639bd1f89dbcb9feac2d984.json](https://raw.githubusercontent.com/ByMykel/counter-strike-price-tracker/main/static/pricehistory/6aea523ef0fce5b5b639bd1f89dbcb9feac2d984.json)

Again this might not be the best way to provide this data but it's free.

**Comment by VladimirHumeniuk** [comment]
<img width="1892" height="962" alt="cs2dex.gg home page" src="https://github.com/user-attachments/assets/241d24ce-adcf-402c-9a53-a9ddc73828eb" />
<img width="1894" height="968" alt="cs2dex.gg inventory analizer" src="https://github.com/user-attachments/assets/5c8a4d68-f902-44ee-982b-cba5e09b2c8a" />

[https://cs2dex.gg ](https://cs2dex.gg )- tradeups calculator with outcome simulator, you can see possible tradeups from your inventory, offers calculator, skins catalogue, and other Counter Strike 2 tools for skins/tradeups/inventory management.
